tendrl/monitoring_integration/upgrades/delete_dashboards.py

```diff
@@ -54,13 +54,11 @@ def main():
     try:
         print ("\n Migrating graphite data \n")
         os.system(
-            "PYTHONPATH=$GRAPHITE_ROOT/webapp/ "
```

Comment:
PYTHONPATH is not required, it contains nothing, As per our custom modified files directories for graphite we don't need to give this. In tendrl-ansible also we are executing without PYTHONPATH to initialize. I forgot to remove here.

Reply:
Ack